### PR TITLE
  🐛 Sort services naturally and adapt test

### PR DESCRIPTION
**Pull request by mguidon**
## What do these changes do?

Fixes bug with auto upgrade of access rights in catalog. (Fixes it but does not make it efficient)

## Related issue/s

- Fixes ITISFoundation/osparc-simcore#2518

### CODE REVIEW

File: services/catalog/requirements/_test.in [hunk 1]
```diff
@@ -15,6 +15,7 @@ jsonschema
 # testing
 pytest
 pytest-aiohttp
+pytest-asyncio
```
Comment (sanderegg) [inline services/catalog/requirements/_test.in:18]: this should not be necessary. look at catalog _test.txt.
pytest-asyncio is a dependency of pytest-aiohttp. Please remove it.

File: services/catalog/src/simcore_service_catalog/db/repositories/services.py [hunk 2]
```diff
@@ -141,7 +141,10 @@ async def list_service_releases(
             async for row in await conn.stream(query):
                 releases.append(ServiceMetaDataAtDB(**row))
 
-        return releases
+        # Now sort naturally from latest first: (This is lame, the sorting should be done in the db)
+        return sorted(
+            releases, key=lambda x: [int(v) for v in x.version.split(".")], reverse=True
```
Comment (sanderegg) [inline services/catalog/src/simcore_service_catalog/db/repositories/services.py:146]: maybe have a look at line 371 (this should be I think the real solution, integrating semver in postgres such that it understands versions).
And for the sorting you could use ```sorted(releases, key=lambda x: packaging.version.parse(x.version)) ``` (not sure about reverse in this case) but it should be more robust to whatever version

**Review by sanderegg — CHANGES_REQUESTED** [review]
finally it's there! thanks for taking care of it. If you can just revert the requiremetns file as it is not necessary.

Also please consider using ```packaging```

File: services/catalog/requirements/_test.in (hunk 1 above)
Comment (mguidon) [reply services/catalog/requirements/_test.in]: done

File: services/catalog/src/simcore_service_catalog/db/repositories/services.py (hunk 2 above)
Comment (mguidon) [reply services/catalog/src/simcore_service_catalog/db/repositories/services.py]: thanks, much cleaner and reverse works

**Review by pcrespov — APPROVED** [review]
:tada: thx a lot for this fix!

this closes #2518

**Review by sanderegg — APPROVED** [review]

**Pull request merged**